src/test/java/com/wcc/platform/controller/AboutControllerTest.java

```diff
@@ -150,8 +150,8 @@ void testCollaboratorOkResponse() throws Exception {
         .andExpect(jsonPath("$.collaborators[0].memberTypes[0]", is("COLLABORATOR")))
```

Comment:
If member types are to be serialized to lower letters as well, then also COLLABORATOR will be changed to lowercase.

Reply:
I don't think we should change member types to lowercase, desktop/mobile and the other types could be used in css classes on frontend that is why I made sure to be lowercase, but we can double check with frontend team if it is necessary. But we could make this change in another PR.